spec/defines/config_spec.rb

```diff
@@ -3,98 +3,129 @@
 require 'spec_helper'
 
 describe 'yum::config' do
-  context 'with no parameters' do
-    let(:title) { 'assumeyes' }
+  on_supported_os.each do |os, facts|
```

Comment:
can you please use:
```suggestion
  on_supported_os.each do |os, os_facts|
```

That makes it easier to distinguish between `os_facts` and the rspec `facts` variable.